### DISCUSSION TITLE
Plugin/Integration/GitHub Action for Remote Server File Access w/ VSCode

### DISCUSSION THREAD

**Discussion by TheLarkInn** [cc-tweaked/CC-Tweaked#1757] [General]
I love using CC:Tweaked and writing Lua inside of VSCode when I'm running a local instance of Create:Above&Beyond, however I'd love to play on some SMP's for the modpack, but not sacrifice the ease at which I develop my programs and routines via VSCode. Has anyone figured out a set of GitHub actions or anything crazy fun that syncs with a minecraft server?

**Comment by SirEdvin** [comment]
You probably need other way around. Like, you running applications at computer should check a newest version of files in your git repository and then download them, if new version exists.
Or you can try to use websockets and build tasks queue for this.

**Comment by Wojbie** [comment]
I believe https://marketplace.visualstudio.com/items?itemName=jackmacwindows.craftos-pc does what you are asking for? It uses https://www.craftos-pc.cc/docs/remote to connect to computer. This video explains how it does it. https://www.youtube.com/watch?v=HtafiCaT6CI
